bottles/frontend/ui/details-bottle.ui

```diff
@@ -118,12 +118,12 @@
                         <property name="valign">center</property>
                         <property name="halign">center</property>
                         <property name="margin-bottom">5</property>
-                        <property name="text" translatable="yes">Run with Arguments…</property>
+                        <property name="text" translatable="yes">Run with addition arguments…</property>
```

Comment:
Shouldn't this be `Run with additional arguments`?

Reply:
Yes it should, I'll fix the typo.